front/public/index.html

```diff
@@ -20,8 +34,15 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <link rel="stylesheet" href="%PUBLIC_URL%/css/styles.css">
+    <link rel="stylesheet" type = "text/css" href="%PUBLIC_URL%/css/styles.css">
     <link rel="stylesheet" href="%PUBLIC_URL%/css/layout.css">
+
+
+ <!-- Google authentication -->
```

Comment:
I thought we were going to make this a react component?

Reply:
Oauth is a react component

Reply:
Oh then why is it in a html file? Just curious

Reply:
o because when anjali sent me hers i was thinking of adding html and i never removed it from the index.html